### DISCUSSION TITLE
Pull quote like this

### DISCUSSION THREAD

**Discussion by kemalgencay** [tailwindlabs/tailwindcss#11426] [General]
Hi, 
I want to do something like this: https://htmldog.com/examples/pullquotes2/
Is it possible with tailwind?
TIA
Regards

**Comment by wongjn** [comment]
Yep! If you can do it with CSS then it can be translated to Tailwind.

**Reply by kemalgencay** [reply]
Thanks, so I have to do it line by line with hand, no automatic conversion?

**Reply by wongjn** [reply]
You might be able to find a tool with an internet search to automatically convert to Tailwind, but there's not a lot of properties to convert so manually converting shouldn't be a daunting task.

**Reply by kemalgencay** [reply]
I'll try, thanks
